DfciPkg/Library/DfciRecoveryLib/DfciRecoveryLib.c

```diff
@@ -22,6 +22,89 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define   RANDOM_SEED_BUFFER_SIZE  64   // We'll seed with 64 bytes of random so that OAEP can work its best.
 
+//
+// These represent UEFI SPEC defined algorithms that should be supported by
+// the RNG protocol and are generally considered secure for DFCI purposes.
+//
+static EFI_GUID *CONST  mSecureRngAlgorithms[] = {
+  &gEfiRngAlgorithmSp80090Ctr256Guid,  // SP800-90A DRBG CTR using AES-256
+  &gEfiRngAlgorithmSp80090Hmac256Guid, // SP800-90A DRBG HMAC using SHA-256
+  &gEfiRngAlgorithmSp80090Hash256Guid, // SP800-90A DRBG Hash using SHA-256
+  &gEfiRngAlgorithmArmRndr,            // unspecified SP800-90A DRBG via ARM RNDR register

```

Comment:
NIT: Technically the ARM site lists the potential support as NIST SP 800-90B compliant.

Reply:
This was taken from a comment in NetworkPkg. I'll update it.